package.json

```diff
@@ -53,6 +53,7 @@
     "@storybook/components": "^6.0.17",
     "@storybook/html": "^6.0.17",
     "@storybook/theming": "^6.0.17",
+    "@storybook/addon-viewport": "^6.0.26",
```

Comment:
Should we maybe use 6.0.17 to be perfectly aligned with the other versions? If you delete the package-lock and recreate it, those will be lifted anyways.

Reply:
done